sc-poc/modules/auth/types/jwt.go

```diff
@@ -7,7 +7,7 @@ type (
 	AuthSecret struct {
 		v1alpha1.AuthSecret
 		Alg   JWTAlg
-		Value string
+		Value interface{}
```

Comment:
Remove Value and instead add privateKey, publicKey fields. for HSA, both fields will have same value.